sunpy/net/dataretriever/attrs/goes.py

```diff
@@ -1,9 +1,18 @@
 from sunpy.net.attr import SimpleAttr
 
-__all__ = ['SatelliteNumber']
+__all__ = ["SatelliteNumber", "VersionData"]
 
 
 class SatelliteNumber(SimpleAttr):
     """
     The GOES Satellite Number
     """
+
+
+class VersionData(SimpleAttr):
```

Comment:
Can you please add it to the class method `register_values`? So that it is known what can be the possible values for `a.VersionData`.

Reply:
I've just added this to the register_values - can you see if what I've written is ok?

Reply:
I have suggested some changes to it.

Reply:
https://github.com/sunpy/sunpy/pull/4394#discussion_r461384748